src/Executables/ExportCoordinates/ExportCoordinates.hpp

```diff
@@ -145,6 +148,22 @@ struct ExportCoordinates {
         db::tag_name<domain::Tags::DetInvJacobian<Frame::ElementLogical,
                                                   Frame::Inertial>>(),
         get(det_inv_jac));
+
+    // Also output the jacobian diagnostic, which compares the analytic
+    // Jacobian (via the CoordinateMap) to the numerical Jacobian
+    // (computed via logical_partial_derivative)
+    const auto& jacobian = determinant_and_inverse(inv_jacobian).second;
```

Comment:
Would it be quick to get the Jacobian into the DataBox instead of inverting the inverse Jac? You'll need this for evolution executable anyway I think.

Reply:
Following discussion on 1/30/23 call, we agreed to leave this as-is for ExportCoordinates and for me to coordinate with @nilsdeppe, @markscheel for where to add it down the road to make this diagnostic available as an AMR criterion